Unmet Peer Dependency, Webpack ^3.0

Hello, Thank you for the great plugin. 

I'm getting this error after installing this plugin through npm

```
npm ERR! peer dep missing: webpack@^2.2.1 | ^3.0.0, required by webpack-chrome-extension-reloader@0.7.0
```

Even though my version of webpack is at `"webpack": "^3.10.0"`. The error signified it could either be `webpack@^2.2.1 | ^3.0.0`, so I'm assuming that `3.10.0` should be also covered? Any help is appreciated. 

<img width="518" alt="screen shot 2018-02-06 at 5 21 12 pm" src="https://user-images.githubusercontent.com/5641734/35893340-795d0b08-0b62-11e8-9ed8-c0e37e3057ec.png">

--- a/package.json
+++ b/package.json
@@ -44,7 +44,7 @@
     "ws": "^3.3.3"
   },
   "peerDependencies": {
-    "webpack": "^2.2.1 | ^3.0.0"
+    "webpack": "2 | 3"
   },
   "devDependencies": {
     "@types/chai": "^4.0.0",

